.github/workflows/release.yml

```diff
@@ -239,12 +239,7 @@ jobs:
       - name: Create VERSION.TXT
         shell: bash
         run: |
-          cat >VERSION.txt <<EOL
-          build type: ${{ matrix.artifact }}
-          build number: ${{ needs.release.outputs.timestamp }}
-          commit sha: ${{ github.sha }}
-          commit url: https://github.com/${{ github.repository }}/commit/${{ github.sha }}
-          EOL
+          python3 build-scripts/version.py ARTIFACT=${{ matrix.artifact }} TIMESTAMP=${{ needs.release.outputs.timestamp }}
```

Comment:
Maybe leave this one be? It's not doing any sort of git invocations, or any complex logic.
It's only needed for release, not in the normal dev workflow, there should be no need to move it into python, right? It would make the script more lightweight too.

Reply:
True, but I would prefer keeping the option that a release build can be done offline without GHA.

Reply:
(test test, i'm curious how github would handle this one)